lib/core.sh

```diff
@@ -5,9 +5,9 @@ KYRAT_TMPDIR=${KYRAT_TMPDIR:-/tmp}
 KYRAT_SHELL=${KYRAT_SHELL:-bash}
 
 BASE64=base64
-BASH=bash
-ZSH=zsh
-SH=sh
+BASH=${KYRAT_BASH:-bash}
+ZSH=${KYRAT_ZSH:-zsh}
+SH=${KYRAT_SH:-sh}
```

Comment:
Whenever possible, I'd like to reduce environment variables if not needed. For your specific case, is it enough to update the `KYRAT_SHELL` only?

```
KYRAT_SHELL=/Users/user/.nix-profile/bin/bash ssh darwin
```

Reply:
Then it complains about:

```
KYRAT_SHELL=/Users/user/.nix-profile/bin/bash command kyrat darwin
KYRAT_SHELL not set correctly: /Users/user/.nix-profile/bin/bash. Aborting.
```

Not sure how to solve this correct.

Reply:
Ah I know why. That's because, depending on the shell type (zsh, bash) the options to pass to pass to the shell can be different. The options are set here: https://github.com/fsquillace/kyrat/blob/master/lib/core.sh#L157

Another way to fix this, is to let user change the `KYRAT_SHELL_CMD` directly instead. I think for now is fine to use your approach too.

Reply:
> Another way to fix this, is to let user change the KYRAT_SHELL_CMD directly instead. I think for now is fine to use your approach too.

I wanted to do this initially but it required to much code changes which I did not want to do.